SimpleMvvmToolkit.Samples.Messaging.Core/SimpleMvvmToolkit.Samples.Messaging.Core/App.xaml.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
```

Comment:
Update to add DI code as in [SimpleMvvmToolkit.Samples.WPF.Core](https://github.com/SimpleMvvm/SimpleMvvmToolkit.Samples.WPF.Core).